src/en/spmanhwa/src/eu/kanade/tachiyomi/extension/en/spmanhwa/Spmanhwa.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.en.spmanhwa
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Spmanhwa : Madara(
+    "Spmanhwa",
+    "https://spmanhwa.online",
+    "en",
+    SimpleDateFormat("MMMMM dd, yyyy", Locale("en")),
```

Comment:
the default date format in madara should work no?

Reply:
Yes, I saw the date format of the source, but I didn't see the default date format of the theme